### DISCUSSION TITLE
Unable to access Github Education benefits despite already being approved

### DISCUSSION THREAD

**Discussion by Nazrin99** [community/community#143549] [GitHub Education]
### Hi everyone,

Hi, my application for Github Education has been approved and it says at my application "Congrats! Your request for [@Nazrin99](https://github.com/Nazrin99) was approved and your benefits are now available." and the badge turned purple. But I can't seem to access Github Copilot in Intellij IDEA. It says that my subscription is not found. What should I do?
![Screenshot 2024-11-05 004919](https://github.com/user-attachments/assets/346e8242-418f-4089-83d9-0c241396a7fa)

**Comment by davevad93** [comment]
Hi @Nazrin99 ,

You should have access to your benefits indeed, try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it and get access to `Copilot` in `Intellij IDEA` you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select 
"`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
